src/types/cells.ts

```diff
@@ -25,6 +24,7 @@ export type Cell = LiteralCell | FormulaCell;
 
 interface EvaluatedCellProperties {
   readonly format?: Format;
+  readonly message?: string;
```

Comment:
I don't think we should have `message` here since it's specific to a cell value and we do have a specific type for it.
```suggestion
```